calliope/test/test_analysis.py

```diff
@@ -48,9 +48,30 @@ def test_national_scale_plotting(self, national_scale_example):
                 plotly_kwarg_updates={'auto_open': False, 'filename': out_path}
             )
 
+    @python36_or_higher
```

Comment:
not clear what this test is adding

Reply:
It has bi-directional flow, so there's some functionality in `flows` which would otherwise not be triggered (neither national scale nor milp models have bi-directional flow, the latter due to the introduction of the binary operation switch). I think plotting tests could be cleaned up generally, but this was a quick (and dirty?) way of keeping coverage up ;)